src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -257,4 +257,8 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System.Management" />
```

Comment:
Is this needed? Is this for the registry stuff? Then please add it to the netdesktop item group